lib/discordrb/data/channel.rb

```diff
@@ -994,6 +1016,7 @@ def update_data(new_data = nil)
       @parent_id = new_data[:parent_id] || new_data['parent_id'] || @parent_id
       process_permission_overwrites(new_data[:permission_overwrites] || new_data['permission_overwrites'])
       @rate_limit_per_user = new_data[:rate_limit_per_user] || new_data['rate_limit_per_user'] || @rate_limit_per_user
+      process_last_pin_timestamp(new_data['last_pin_timestamp']) if new_data['last_pin_timestamp']
```

Comment:
I wonder what happens if there's only one pin and then we unpin, should we be checking for key presence rather than key value truthiness?